programs/fio/parse

```diff
@@ -43,7 +43,7 @@ OPS.each do |ops|
       sp = p.chomp('0')
       true while sp.chomp!('0')
       sp.chomp!('.')
-      puts "#{ops}_clat_#{sp}%_us: #{res_clat_percentiles[p]}"
+      puts "#{ops}_clat_#{sp}%_ns: #{res_clat_percentiles[p]}"
```

Comment:
thanks, would you help also update below code to use ns, and resolve rspec failure (refer to the failure of Checks which can b reproduced locally by rake spec spec=stats).
```
  puts "#{ops}_clat_mean_us: #{res_clat['mean']}"
```

Reply:
updated, along with fio spec yamls